mint.json

```diff
@@ -127,118 +126,139 @@
-                "integrations/providers/sms/plivo",
-                "integrations/providers/sms/telnyx",
-                "integrations/providers/sms/termii",
-                "integrations/providers/sms/firetext",
-                "integrations/providers/sms/gupshup",
-                "integrations/providers/sms/clickatell"
+                "workflow/channels/email",
+                {
+                  "group": "Providers",
+                  "pages": [
+                    "workflow/channels/email/sendgrid",
+                    "workflow/channels/email/amazon-ses",
+                    "workflow/channels/email/postmark",
+                    "workflow/channels/email/resend",
+                    "workflow/channels/email/sendinblue",
+                    "workflow/channels/email/mailersend",
+                    "workflow/channels/email/mailgun",
+                    "workflow/channels/email/mailjet",
+                    "workflow/channels/email/infobip",
+                    "workflow/channels/email/mailtrap",
+                    "workflow/channels/email/mandrill",
+                    "workflow/channels/email/netcore",
+                    "workflow/channels/email/outlook365",
+                    "workflow/channels/email/braze",
+                    "workflow/channels/email/sparkpost",
+                    "workflow/channels/email/plunk",
+                    "workflow/channels/email/custom-smtp"
+                  ]
+                }
               ]
-            }
-          ]
-        },
-        {
-          "group": "Push",
-          "icon": "mobile",
-          "pages": [
-            "integrations/providers/push/overview",
+            },
+            {
+              "group": "Push",
+              "icon": "mobile",
+              "pages": [
+                "workflow/channels/push",
+                {
+                  "group": "Providers",
+                  "pages": [
+                    "workflow/channels/push/fcm",
+                    "workflow/channels/push/apns",
+                    "workflow/channels/push/expo-push",
+                    "workflow/channels/push/onesignal",
+                    "workflow/channels/push/pushpad",
+                    "workflow/channels/push/push-webhook",
+                    "workflow/channels/push/pusher-beams"
+                  ]
+                }
+              ]
+            },
             {
-              "group": "Providers",
+              "group": "SMS",
+              "icon": "message-sms",
               "pages": [
-                "integrations/providers/push/fcm",
-                "integrations/providers/push/apns",
-                "integrations/providers/push/expo-push",
-                "integrations/providers/push/onesignal",
-                "integrations/providers/push/pushpad",
-                "integrations/providers/push/push-webhook",
-                "integrations/providers/push/pusher-beams"
+                "workflow/channels/sms",
```

Comment:
Seems like you changed all the urls for those pages, it will affect SEO. We will need to have redirects to them. But, I don't think the delivery providers should live under workflow, would keep it as parallel concept